scripts/update_metadata.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+'''
+@author: Patrick Cleeve
+
+This is a script to update the MD_POS_COR for the stage-bare component while odemis is running.
```

Comment:
You should at least indicate that it's for the METEOR, on TFS SEMs. Maybe also explain a bit why someone would want to change that value.